package.json

```diff
@@ -5,14 +5,17 @@
   "repository": "https://github.com/imgix/imgix-core-js",
   "scripts": {
     "assert_version": "node assert_version.js",
-    "compile": "mkdir -p dist && uglifyjs ./src/imgix-core-js.js -o ./dist/imgix-core-js.min.js && cp ./src/imgix-core-js.js ./dist/imgix-core-js.js",
+    "compile": "mkdir -p dist && uglifyjs ./src/imgix-core-js.js -o ./dist/imgix-core-js.min.js && cp ./src/imgix-core-js.js ./dist/imgix-core-js.js && cp ./src/imgix-core-js.d.ts ./dist/imgix-core-js.d.ts && tsc",
     "prepublish": "npm run compile && npm run assert_version",
     "test": "mocha"
   },
   "main": "dist/imgix-core-js.js",
+  "types": "src/",
```

Comment:
Should this be pointing to a specific file, or is it correct to point to the whole directory?

Reply:
Should point to `src/imgix-core-js.d.ts` in this case as per https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package

Reply:
> Set the `types` property to point to your bundled declaration file.

Hmm so it would actually have to point to `dist/imgix-core-js.d.ts` in that case, no?

Reply:
Oh yea, good call